### PR TITLE
Update @stoe's slide

### PR DESCRIPTION
**Pull request by stoe**
I'm a ninja ※\(^o^)/※

### CODE REVIEW

**Review by rebinsaid — COMMENTED** [review]
Nice feature

File: _slides/03-stoe.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+I'm a Ninja! ※\(^o^)/※
```
Comment (rebinsaid) [inline _slides/03-stoe.md:6]: This is great work! 🥇

Comment (gusshawstewart) [inline _slides/03-stoe.md:6]: This is great work! 😃

Comment (stoe) [reply _slides/03-stoe.md:6]: Thank you @rebinsaid
🐙🐱💖

File: _slides/03-stoe.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (gusshawstewart) [inline _slides/03-stoe.md]:
```suggestion
title: "Welcome to our slide deck, captain!"
```

Comment (rebinsaid) [inline _slides/03-stoe.md]:
```suggestion
title: "Welcome to our slide deck, rebin!"
```

**Review by gusshawstewart — APPROVED** [review]
This is brilliant! well done! 🏅

**Pull request merged**